examples/Tables/Table.md

```diff
@@ -202,6 +202,76 @@ import {
   </TableBody>
 </Table>
 ```
+The following example of the `Table` component is  where `isSticky` prop is set to make the TableHead position sticky.  In this case, when the page is scrolled, the TableHead will stick and the TableHead position defaults to `isSticky` property.
+
+```jsx
+import {
+  ALIGN,
+  Table,
+  TableCell,
+  TableBody,
+  TableRow,
+  TableHeadingCell,
+  TableHead,
+} from 'mark-one';
+
+<Table>
+  <TableHead isSticky>
+    <TableRow>
+      <TableHeadingCell scope={'col'}>ID</TableHeadingCell>
+      <TableHeadingCell scope={'col'}>First Name</TableHeadingCell>
+      <TableHeadingCell scope={'col'}>Last Name</TableHeadingCell>
+    </TableRow>
+  </TableHead>
+  <TableBody isScrollable={true}>
+    <TableRow isStriped={true}>
+      <TableCell>1</TableCell>
+      <TableCell>Kristin</TableCell>
+      <TableCell>Glenn</TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>2</TableCell>
+      <TableCell>Jack</TableCell>
+      <TableCell>Thompson</TableCell>
+    </TableRow>
+    <TableRow isStriped={true}>
+      <TableCell>3</TableCell>
+      <TableCell>Lianne</TableCell>
+      <TableCell>Michaels</TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>4</TableCell>
+      <TableCell>Gabriela</TableCell>
+      <TableCell>Hines</TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>5</TableCell>
+      <TableCell>Hines</TableCell>
+      <TableCell>Gabriela</TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>6</TableCell>
+      <TableCell>Michaels</TableCell>
+      <TableCell>Lianne</TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>7</TableCell>
+      <TableCell>Thompson</TableCell>
+      <TableCell>Jack</TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>8</TableCell>
+      <TableCell>Glenn</TableCell>
+      <TableCell>Kristin</TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>9</TableCell>
+      <TableCell>Gabriela</TableCell>
+      <TableCell>Hines</TableCell>
+    </TableRow>
```

Comment:
Thanks for adding this extended example. I think it's worth generating *a lot* of rows (i.e. in the hundreds) for this, to really show where the feature is useful. We don't need the data to be particularly meaningful, so it's fine to just fill with a sequence of numbers in different columns.

Obviously we don't want to create those by hand, but we can use the second argument to  [`Array.from()`][mdn] method (which is essentially the same as `Array.map()`) to generate them for us, e.g.:

```ts
// To generate a list from 0 to 9
<ol>
  {Array.from({length: 10}, (_, index) => (
    <li>{index}</li>)
  )}
</ol>
```

We should also make sure to extend the row striping all the way down. It may also be helpful to create a simple wrapper component around the table, with absolute positioning, scrolling overflow, and a fixed value for the height. (You can always import the `styled` tool here and define a separate `styled.div` directly in the example body that we can use as a wrapper).

Basically, the goal is just to make sure that our style guide resembles our real use cases as much as possible, both as a reminder for the future and to help us catch any issues in the process. Let me know if you need any help getting that set up!